lomap/algorithms/product.py

```diff
@@ -689,14 +693,14 @@ def markov_times_markov(markov_tuple):
     stack=[]
 
     # Find the initial states of the MDP
-    for init_state in it.product(*map(lambda m: m.init.keys(), markov_tuple)):
+    for init_state in it.product(*[list(m.init.keys()) for m in markov_tuple]):
```

Comment:
The `list` transformation is not needed. Change code to `for init_state in it.product(*[m.init.keys() for m in markov_tuple]):`